vendor/wp-now/src/execute-wp-cli.ts

```diff
@@ -16,11 +18,13 @@ export async function executeWPCli ( projectPath: string, args: string[] ): Prom
 		wp: DEFAULT_WORDPRESS_VERSION,
 		path: projectPath,
 	});
-	const { phpInstances, options: wpNowOptions } = await startWPNow({
-		...options,
-		numberOfPhpInstances: 2,
+	const php: NodePHP = await NodePHP.load(options.phpVersion,{
+		requestHandler: {
+			documentRoot: options.documentRoot,
+			absoluteUrl: options.absoluteUrl,
+		}
 	});
-	const [, php] = phpInstances;
+	php.mount(projectPath, options.documentRoot);
```

Comment:
Improve `wp-cli` performance by avoiding starting the site. In theory, we only need a PHP instance.